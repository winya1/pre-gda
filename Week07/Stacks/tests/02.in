a 
peek 
pop 
b 
c 
peek 
d 
pop
pop
peek
pop
pop
