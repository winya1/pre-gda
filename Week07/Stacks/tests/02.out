1
a
a
1
2
c
3
d
c
b
b
SU
